docs/src/documentation-pages/dev/quickstart.md

```diff
@@ -41,15 +41,13 @@ This quickstart describes the initial setup required to run an instance of Cross
 
 6. Navigate to [http://localhost](http://localhost) in a browser.
 
-7. Create an account on the [staging Crossfeed instance](https://staging.crossfeed.cyber.dhs.gov/signup?). You can then log into your account running on localhost using this account.
-
-8. Hot reloading for source files is enabled, but after changes to non-source code files stopping and starting Docker Compose is required. The following are examples of changes that will require restarting the environment:
+7. Hot reloading for source files is enabled, but after changes to non-source code files stopping and starting Docker Compose is required. The following are examples of changes that will require restarting the environment:
```

Comment:
How should the user create their account then? Maybe add a step that they can create an account from http://localhost/signup ?